### PR TITLE
Minor Shitmed Bugfixes

### PR DESCRIPTION
**Pull request by gluesniffler**
## About the PR
Fixed some bugs I overlooked/missed/forgor. Soon I'll kill the disappearing torsos and the broken slots, pinky promise.

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Mocho
- fix: Fixed borg legs not letting you move properly. 
- fix: Fixed medical cyborgs poisoning people.
- fix: Fixed cybernetic legs being weak to stepping on glass
- fix: Fixed biosynthetic legs for real this time (lol)

## Release Notes

- **New Features**
	- Introduced the `SanitizedComponent` to enhance surgical safety by preventing toxin damage during procedures.
	- Added new cybernetic entities and components, improving movement and interaction capabilities.
	- Enhanced visibility of health information for medical and syndicate cyborgs.

- **Improvements**
	- Streamlined surgery step validation and event handling for better user experience.
	- Updated bio-synthetic entities to include detailed components for arms and legs.

- **Bug Fixes**
	- Resolved issues with sprite paths for pizza-themed entities for consistency.

### CODE REVIEW

File: Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml [hunk 1]
```diff
@@ -275,7 +275,8 @@
     interactSuccessSound:
       path: /Audio/Ambience/Objects/periodic_beep.ogg
   - type: SurgeryTarget
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml:277]: pro remove this from borg

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Aidenkrz** [comment]
yaml linter failure, thats so embarassing

**Comment by gluesniffler** [comment]
> yaml linter failure, thats so embarassing

should've seen me not deleting some of the autocomplete suggestions from my IDE on the syringe gun PR lmao

File: Resources/Prototypes/_Shitmed/Body/Parts/generic.yml [hunk 2]
```diff
@@ -27,12 +39,24 @@
   id: BioSynthLeftLeg
   name: bio-synthetic left leg
   description: This left leg can be transplanted into any living organism and it will adapt to its recipient.
+  components:
+  - type: BodyPart
+    children:
+      right foot:
+        id: "right foot"
+        type: Foot
```
[the remark this answers is not in the document]
Comment (gluesniffler) [reply Resources/Prototypes/_Shitmed/Body/Parts/generic.yml:47]: man i shouldnt be coding at 4am huh

File: Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml (hunk 1 above)
Comment (gluesniffler) [reply Resources/Prototypes/Entities/Mobs/Cyborgs/borg_chassis.yml:277]: ill nuke it when your PR is merged

**Pull request merged**